setup.py

```diff
@@ -15,7 +15,7 @@
     author='Boris Skurikhin',
     author_email='boriskurikhin@gmail.com',
     url='https://github.com/boriskurikhin/pyuniqid',
-    download_url='https://github.com/boriskurikhin/pyuniqid/archive/v1.2.tar.gz',  # noqa
+    download_url='https://test.123',
```

Comment:
This was left in by mistake, but when you think about it it doesn't really matter because it will be replaced in the `envsubst` step.

Nonetheless I'll change it in a new commit.